horizons/gui/ingamegui.py

```diff
@@ -454,7 +458,7 @@ def on_key_press(self, action, evt):
 		keyval = evt.getKey().getValue()
 
 		if action == _Actions.ESCAPE:
-			return self.on_escape()		
```

Comment:
I can confirm, I've had stripped whitespace between commits as well. Perhaps we should push a single commit removing trailing whitespace and be done with it.